src/http/endpoints/update_profile.rs

```diff
@@ -0,0 +1,86 @@
+use crate::ayb_db::db_interfaces::AybDb;
+use crate::ayb_db::models::{Entity, InstantiatedEntity, Link};
+use crate::error::AybError;
+use crate::http::structs::EntityPath;
+use crate::http::utils::unwrap_authenticated_entity;
+use crate::http::verification::is_verified;
+use crate::http::web_frontend::WebFrontendDetails;
+use actix_web::{post, web, HttpResponse};
+use serde::Deserialize;
+use std::str::FromStr;
+use url::Url;
+
+#[derive(Deserialize)]
```

Comment:
So nice to have this endpoint-specific struct stuff in the same file as the endpoint. I'm glad you cleaned up our endpoints earlier! :)

Reply:
:)